src/components/HomeHeader.tsx

```diff
@@ -0,0 +1,25 @@
+import Subtitle from './Subtitle';
+import LocationSelect from './LocationSelect';
+import { Location } from '../types/sparkeats';
+
+type HomeHeaderProps = {
+  locations: Location[];
+};
+
+function HomeHeader({ locations }: HomeHeaderProps) {
+  return (
+    <div className="home-header">
+      <Subtitle />
+
+      <LocationSelect locations={locations} />
+
+      <div className="home-header__button">
+        <a className="button__header-primary" href="/places/new">
+          Add a Review
```

Comment:
```suggestion
          Add a Location
```

This should change because it will go to `/locations/new` not `/reviews/new`.

Reply:
Updated in d69acaf2d50d6cebdcbec0e5865412849eddb683.